<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QDialog" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>794</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="nameLBL">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="Line_Name">
         <property name="maximumSize">
          <size>
           <width>16777</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maxLength">
          <number>2500</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="ageLBL">
         <property name="text">
          <string>Age</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Line_Age"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="heightLBL">
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="Line_Height"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="weightLBL">
         <property name="text">
          <string>Weight</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Line_Weight"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Somatotyp</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="CBX_Soma">
         <item>
          <property name="text">
           <string>Choose somatotyp</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ektomorph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mezomorph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ektomezomorph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Endomezomorph</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Endomorph</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Line_BF"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Body fat</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="genderBTN_M">
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="genderBTN_FM">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Activity</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="activityBTN_No">
         <property name="font">
          <font>
           <fontweight>DemiBold</fontweight>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>No activity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="activityBTN_Light">
         <property name="text">
          <string>Light activity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="activityBTN_Middle">
         <property name="text">
          <string>Middle activity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="activityBTN_Intense">
         <property name="text">
          <string>Intensive activity</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Goal</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QRadioButton" name="goalBTN_Lose">
         <property name="text">
          <string>Lose weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="goalBTN_Gain">
         <property name="text">
          <string>Gain weight</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="goalBTN_Maintain">
         <property name="text">
          <string>Weight maintenaince</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Type of training</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QRadioButton" name="trainingBTN_BW">
         <property name="text">
          <string>Bodyweight training</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="trainingBTN_STR">
         <property name="text">
          <string>Strenght training</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSubmit">
     <property name="font">
      <font>
       <family>Lucida Sans Unicode</family>
      </font>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_picture_1">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
